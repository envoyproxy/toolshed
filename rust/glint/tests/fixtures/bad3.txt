This file only has trailing whitespace issues   
Line 2 also has trailing spaces  
Line 3 is clean
Line 4 has a tab at the end	
Line 5 is clean
Line 6 has mixed trailing whitespace 	 
Final line with newline
