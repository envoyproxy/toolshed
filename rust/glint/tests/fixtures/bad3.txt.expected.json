{
  "files": {
    "tests/fixtures/bad3.txt": {
      "trailing_whitespace": [
        1,
        2,
        4,
        6
      ]
    }
  },
  "summary": {
    "total_files": 1,
    "files_with_issues": 1,
    "total_issues": 4
  }
}
